staging_service/app.py

```diff
@@ -108,31 +104,45 @@ def _file_type_resolver(path: PathPy) -> FileTypeResolution:
         return FileTypeResolution(unsupported_type=ext)
 
 
+def _make_dts_file_resolver() -> Callable[[Path], FileTypeResolution]:
```

Comment:
This looks very silly right now, but it's a separate function to allow parameterizing the parser. Next PR will make use of a JSON schema file that'll need to get loaded up and passed to the DTS parser.